torch/fx/experimental/proxy_tensor.py

```diff
@@ -187,7 +189,13 @@ def __new__(cls, elem, proxy, *, requires_grad=None):
         def create_proxy_symint(sym_int, new_proxy):
             return torch._C.SymbolicIntNode.new_symint(ProxySymInt(sym_int, new_proxy))
 
-        r = torch.Tensor._make_wrapper_subclass(cls, [create_proxy_symint(elem.shape[i], proxy.size(i)) for i in range(len(elem.shape))], dtype=elem.dtype, layout=elem.layout, device=elem.device, requires_grad=elem.requires_grad, strides=create_contiguous(elem.shape), storage_offset=elem.storage_offset())
+        r = torch.Tensor._make_wrapper_subclass(
+            cls, [
+                create_proxy_symint(elem.shape[i], proxy.size(i)) for i in range(len(elem.shape))
+            ], dtype=elem.dtype, layout=elem.layout, device=elem.device, requires_grad=elem.requires_grad,
+            strides=[
+                create_proxy_symint(stride_i, proxy.stride(i)) for i, stride_i in enumerate(create_contiguous(elem.shape))
+            ], storage_offset=elem.storage_offset())
```

Comment:
@Chillee I wonder if this is a footgun. Before, the symbolic sizes code here had a proxy associated with it but the strides didn't. So in C++ if you try to do something like `mul(size[i], stride[i])`, I ran into trouble because the two backing objects are now `ProxySymInt` and `PySymInt`, which ends up surfacing as an error inside of `fx.Proxy`.